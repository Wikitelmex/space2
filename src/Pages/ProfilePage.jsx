import React from 'react';
import MyMissionsComponent from '../Components/MyMissionsComponent';
import MyRockets from '../Components/MyRockets';

const ProfilePage = () => {
  const PageName = <h1>Profile Page</h1>;

  return (
    <div>
      {PageName}
      <div className="d-flex flex-row col-12">
        <div className="col-6 p-3">
          <h2>My Missions</h2>
          <MyMissionsComponent />
        </div>
        <div className="col-6 p-3">
          <h2>My Rockets</h2>
          <MyRockets />
        </div>
      </div>
    </div>
  );
};

export default ProfilePage;
