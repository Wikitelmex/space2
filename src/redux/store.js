import { createStore, applyMiddleware, combineReducers } from 'redux';
import logger from 'redux-logger';
import thunk from 'redux-thunk';

// import reducers here
import missionsReducer from './missions/missionReducer.js';

const mixReducer = combineReducers({
  // add reducers here
  missions: missionsReducer,
});

const store = createStore(
  mixReducer,
  applyMiddleware(logger, thunk),
);

export default store;
