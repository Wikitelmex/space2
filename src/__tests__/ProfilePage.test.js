import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import ProfilePage from '../Pages/ProfilePage';
import store from '../redux/store';

test('matches ProfilePage component snapshot', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <ProfilePage />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
