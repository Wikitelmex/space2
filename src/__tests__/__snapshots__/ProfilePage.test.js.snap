// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches ProfilePage component snapshot 1`] = `
<div>
  <h1>
    Profile Page
  </h1>
  <div
    className="d-flex flex-row col-12"
  >
    <div
      className="col-6 p-3"
    >
      <h2>
        My Missions
      </h2>
      <div>
        No Missions loaded yet...
      </div>
    </div>
    <div
      className="col-6 p-3"
    >
      <h2>
        My Rockets
      </h2>
      <div>
        <table
          className="border col-12"
        />
      </div>
    </div>
  </div>
</div>
`;
